CHANGELOG.md

```
@@ -1,3 +1,8 @@
+## 0.0.7
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)